src/container/cli.c

```diff
@@ -2687,12 +2716,12 @@ dc_cont_list_snap(tse_task_t *task)
 		*args->nr = 0;
 
 	rc = cont_req_prepare(args->coh, CONT_SNAP_LIST,
-			     daos_task2ctx(task), &cb_args);
+			      daos_task2ctx(task), &cb_args);
 	if (rc != 0)
 		D_GOTO(out, rc);
 
 	D_DEBUG(DF_DSMC, DF_CONT": listing snapshots: hdl="
-			 DF_UUID "; size=%d\n",
+			 DF_UUID"; size=%d\n",
```

Comment:
This is using ; where most cases seem to use :
```suggestion
			 DF_UUID": size=%d\n",
```

Reply:
ok